README.md

```diff
@@ -14,11 +14,25 @@ For local development you can simulate lambda and an API endpoint locally using
 the following command.
 
 ```bash
-serverless offline start
+yarn offline
 ```
 
 This repository uses JavaScript import aliases with Webpack. If you are using VS Code, the `jsconfig.json` file should handle the autocompletion for you. If you are using WebStorm, you can refer to [this stackoverflow thread](https://stackoverflow.com/questions/34943631/path-aliases-for-imports-in-webstorm).
 
+## Debugging (VS Code)
+
+This repo includes a pre-configured template for the VS Code NodeJS Debugger. It can be found at `.templates/vscode/launch.json`.
+
+To get started with debugging, copy the file to `.vscode/launch.json`, or copy its contents to your existing `.vscode/launch.json` file.
+
+Now you can press `CTRL+SHIFT+D` to open the `Run` tab and execute the `Debug: Serverless Offline` configuration.
```

Comment:
Seems to be SHIFT+CMD+D, on Mac. Is CTRL on Windows? https://code.visualstudio.com/docs/editor/debugging

Reply:
Yeah. Windows and Linux. Thanks Corin!

Reply:
79cc0a526274506be6a07883a9f2d3638d62921b